test/jasmine/tests/select_test.js

```diff
@@ -130,88 +135,6 @@ describe('Test select box and lasso in general:', function() {
         });
     }
 
-    describe('select elements', function() {
-        var mockCopy = Lib.extendDeep({}, mock);
-        mockCopy.layout.dragmode = 'select';
-
-        var gd;
-        beforeEach(function(done) {
-            gd = createGraphDiv();
-
-            Plotly.plot(gd, mockCopy.data, mockCopy.layout)
-                .then(done);
-        });
-
-        it('should be appended to the zoom layer', function(done) {
-            var x0 = 100,
-                y0 = 200,
-                x1 = 150,
-                y1 = 250,
-                x2 = 50,
-                y2 = 50;
```

Comment:
`(x2, y2)` is in the top margin of this plot, so the doubleclick didn't do anything. But since these tests didn't actually check that the handlers ever got called, it didn't cause a failure.

Reply:
Nice investigation work. Thanks!